ETC.tsv

```diff
@@ -4340,7 +4340,7 @@ ETC_20150317_004339	Requires iCoin.{nl}Are you sure you want to reset?
 ETC_20150317_004340	{@st59} Increase in Physical Attack and Critical Attack{/}
 ETC_20150317_004341	{@st59} Increase in HP, HP Recovery rate, Critical Resistance and Max Inventory Slots{/}
 ETC_20150317_004342	{@st59} Increase in Magic Damage and Block Penetration{/}
-ETC_20150317_004343	{@st59} Increase in Accuracy, Evasion and Critical Attack {/}
+ETC_20150317_004343	{@st59} Increase in Accuracy, Evasion and Critical Rate {/}
```

Comment:
That's already part of #871 
Please tell me when you include my fixes so that I don't keep a separate PR.